src/udev/udevadm-hwdb.c

```diff
@@ -646,9 +651,12 @@ static int adm_hwdb(struct udev *udev, int argc, char *argv[]) {
                 }
                 STRV_FOREACH(f, files) {
                         log_debug("reading file '%s'", *f);
-                        import_file(udev, trie, *f);
+                        if (import_file(udev, trie, *f) < 0)
+                            rc = EXIT_FAILURE;
                 }
                 strv_free(files);
+                if (!rc)
```

Comment:
This condition seems to be reversed.

Also, please use an explicit comparison, e.g. `rc != 0`.

Reply:
It's correct. But replaced by `rc != EXIT_SUCCESS`.